getstate.go

```diff
@@ -5,14 +5,16 @@ import "context"
 type GetStateRequest struct {
 	BaseRequest
 
-	PaymentID string `json:"PaymentId"`    // Идентификатор платежа в системе банка. По офф. документации это number(20), но фактически значение передается в виде строки
-	ClientIP  string `json:"IP,omitempty"` // IP-адрес покупателя
+	PaymentID   string `json:"PaymentId"`    // Идентификатор платежа в системе банка. По офф. документации это number(20), но фактически значение передается в виде строки
+	TerminalKey string `json:"TerminalKey"`  // Ключ терминала
+	ClientIP    string `json:"IP,omitempty"` // IP-адрес покупателя
 }
 
 func (i *GetStateRequest) GetValuesForToken() map[string]string {
 	return map[string]string{
-		"IP":        i.ClientIP,
-		"PaymentId": i.PaymentID,
+		"IP":          i.ClientIP,
+		"PaymentId":   i.PaymentID,
+		"TerminalKey": i.TerminalKey,
```

Comment:
This field is added automatically in `Client.secureRequest()`

Reply:
![изображение](https://github.com/nikita-vanyasin/tinkoff/assets/68816136/e3c1c7a2-13a2-45fd-8e3e-f90822ca164f)

I cant do request without this option. It writes that they dont have any Terminal Key in the request (Check the couple of terminal key and secret key)

Reply:
Oh looks like there is a flaw in the design of this package API. Usage of BaseRequest fields is not very straightforward. It probably can be improved for next major version of the package.


Still your code should be working fine, if you are using proper terminal key + password pair.
Can you do other requests with this `u.tinkoffClient`?
Can you show how do you initialize `u.tinkoffClient`?
You can check your test terminal key/password with this package test suite:
```
TERMINAL_KEY=<DEMO_TERMINAL_KEY> TERMINAL_PASSWORD=<DEMO_TERMINAL_PWD> go test -v ./...
```

Reply:
Terminal and password key is correct because init purchase is working. My code works with my added field like TerminalKey and I clearly input it. I init tinkoffClient like that 
![изображение](https://github.com/nikita-vanyasin/tinkoff/assets/68816136/8f6540f6-db9c-43ce-bc1a-3222299d728a)
and it works

Reply:
@flew1x so far I can not reproduce the issue. Can you please check if go test suite is working for your demo terminal key / password pair? (snippet above)

Is it possible that you are overriding `tinkoffClient.Config` field somehow before calling `GetState`?